### PR TITLE
Add the first Changelog

### PR DESCRIPTION
**Pull request by Gomez**
Fixes:
https://github.com/owncloud/mail/issues/1282

@owncloud/mail

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Changelog
```
Comment (ChristophWurst) [inline CHANGELOG.md]: use a single hash for the heading and two for sub-headings

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+
+### 0.4 (unreleased)
+
+New:
+
+* Add console command for account creation 
+  Howto: https://github.com/owncloud/mail/pull/1202#issuecomment-168435304
+  [ChristophWurst]
```
Comment (ChristophWurst) [inline CHANGELOG.md]: how about appending the github issues/pr instead of the author?

**Comment by ChristophWurst** [comment]
(I aborted travis – there's nothing to test)

File: CHANGELOG.md (hunk 2 above)
Comment (jancborchardt) [inline CHANGELOG.md]: Yeah, issue number makes more sense, then people can check what was done. :)

**Comment by Gomez** [comment]
Updated

**Comment by Gomez** [comment]
Again updated

**Comment by ChristophWurst** [comment]
:+1:

**Pull request merged**